src/common/tests/mock/graphql/mocks.ts

```diff
@@ -0,0 +1,80 @@
+import * as Factory from 'factory.ts';
```

Comment:
we have enabled `esModuleInterop` the import style no need asterik

Reply:
fix here 4c2b35fe55cbf765fc96fe1122a38fafeae7722b

Reply:
actually fix here e0aa3823e50675c5564185f248da32e44d18b219